tracee-ebpf/go.sum

```diff
@@ -1,6 +1,6 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
-github.com/aquasecurity/libbpfgo v0.1.1 h1:j/ipxI/l20lu5Tosn0o9JYj7bD3Cwzi18jIEEu3rMvQ=
-github.com/aquasecurity/libbpfgo v0.1.1/go.mod h1:/+clceXE103FaXvVTIY2HAkQjxNtkra4DRWvZYr2SKw=
+github.com/aquasecurity/libbpfgo v0.1.2-0.20210803032413-c500b6207bf6 h1:5DmiBAntdMSoMC4Be4D8h4cswNnkpkDREiWeluK0BC0=
```

Comment:
Shouldn't this be v0.2.0-libbpf_0.4.0?

Reply:
go mod tidy changes it automagically. I set it as v0.2.0-libbpf_0.4.0 and it changes to this new way of setting versioning.